### ISSUE TITLE
we have same discriminator as sbot

### ISSUE THREAD

**Issue by climu** [climu/OSR-discord-bot#5]
currently we have the same discriminator `!` as raylu's sbot.

It means when users type `!help` it trigger both bots help message.

I wonder if we should change our bot discriminator to avoid this. Maybe `&` or `$`?

**Comment by d-korni** [comment]
I'm not liking the idea of `&`  as much personally, maybe  `$`. And the only issue really comes for the `help` command at the time.

Could we have the `sbot` display the help for `OSR bot` in a second line/post, with the later not having a `help` of its own? I guess this depends on what are the future plans for raylu's `sbot`?

**Comment by climu** [comment]
I think sbot is not only in OSR discord. So we can hardly request changes on it.

Either we get sbot out or we adapt to it.

**Comment by raylu** [comment]
I could implement per-guild discriminators, but it's probably easier to just kick sbot

(but also, patches welcome! https://github.com/raylu/sbot)

**Comment by d-korni** [comment]
Another discriminator option is `.`.
Might be easier for mobile users too.

**Comment by climu** [comment]
I just kicked sbot out of OSR discord. We were not using it very often.

**Comment by climu** [comment]
sbot is now showing twitch go live.
Maybe we could just remove the "command not found" message?

**Comment by raylu** [comment]
yeah, it's pretty weird to respond to everything that starts with a `!` for sure. we should do that regardless
